ng/src/web/pages/tasks/dialog.js

```diff
@@ -362,8 +362,8 @@ const TaskDialog = ({
             <Layout
               flex="column"
               offset="2"
-              className={classes('offset-container',
-                state.in_assets === YES_VALUE ? '' : 'disabled')}>
+              className={'offset-container ' +
```

Comment:
I don't think this is better then before. Maybe we should introduce a new OffsetLayout class? The offset-container css class should be removed in future too.